COMPILING.md

```diff
@@ -282,6 +283,92 @@ To build full curses version with localizations and lua enabled:
 
 Make sure that `x86_64-apple-darwin15-clang++` is in `PATH` environment variable.
 
+## Cross-compile to Android from Linux
+
+The Android build uses [Gradle](https://gradle.org/) to compile the java and native C++ code, and is based heavily off SDL's [Android project template](https://hg.libsdl.org/SDL/file/f1084c419f33/android-project). See the official SDL documentation [README-android.md](https://hg.libsdl.org/SDL/file/f1084c419f33/docs/README-android.md) for further information.
+
+The Gradle project lives in the repository under `android/`. You can build it via the command line or open it in [Android Studio](https://developer.android.com/studio/). For simplicity, it only builds the SDL version with all features enabled, including tiles, sound, localization and lua.
+
+### Dependencies
+
+  * Java JDK 8
+  * SDL2 (tested with 2.0.8, though a custom fork is recommended with project-specific bugfixes)
+  * SDL2_ttf (tested with 2.0.14)
+  * SDL2_mixer (tested with 2.0.2)
+  * SDL2_image (tested with 2.0.3)
+  * libintl-lite (tested with a custom fork of libintl-lite 0.5)
+  * lua (tested with lua 5.1.5)
+  
+Some dependencies need their entire source code copied under the Android project folder so they can be built for Android - see below.
+
+### Setup
+
+Install Linux dependencies. For a desktop Ubuntu installation:
+
+    sudo apt-get install lua5.2 openjdk-8-jdk-headless
+
+Install Android SDK and NDK:
+
+    wget https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip
+    unzip sdk-tools-linux-4333796.zip -d ~/android-sdk
+    rm sdk-tools-linux-4333796.zip
+    ~/android-sdk/tools/bin/sdkmanager --update
+    ~/android-sdk/tools/bin/sdkmanager "tools" "platform-tools" "ndk-bundle"
+    ~/android-sdk/tools/bin/sdkmanager --licenses
+
+Export Android environment variables (you can add these to the end of `~/.bashrc`):
+
+    export ANDROID_SDK_ROOT=~/android-sdk
+    export ANDROID_HOME=~/android-sdk
+    export ANDROID_NDK_ROOT=~/android-sdk/ndk-bundle
+    export PATH=$PATH:$ANDROID_SDK_ROOT/platform-tools
+    export PATH=$PATH:$ANDROID_SDK_ROOT/tools
+    export PATH=$PATH:$ANDROID_NDK_ROOT
+
+Install custom Android dependencies. From the root folder of the CDDA repository run:
```

Comment:
I believe we should switch to `Cataclysm-DDA` folder before cloning.

Reply:
Ah thanks, that's what I meant to say with "From the root folder of the CDDA repository run:" but it's probably clearer to say "Switch to the `Cataclysm-DDA` folder then run:". I'll update that.